skip-file.php

```diff
@@ -97,3 +97,104 @@ function vipgoci_get_skipped_files_issue_message(
 		$affected_files
 	);
 }
+
+/**
+  * IF the file
+ *  Is modified in THIS very commit
+ *  Or New in THIS very commit
+ *  Or Doesn't exist in the previous comments $reached_limit_files
+ *      Let the bot post the comment
+ * ELSE:
+ *     REMOVE FROM the $results
+ *
+ * @param array $pr_issues_results
+ * @param array $comments
+ * @param array $modified_files
+ */
+function vipgo_skip_file_check_pr_comments(array $pr_issues_results = [], array $comments = [], array $modified_files = [])
```

Comment:
`vipgoci_git_diffs_fetch()` can give list of files modified for any two commits.

Reply:
Yep. I saw that one, but It seems that the function requires two SHA to compare A and B, while I was looking for a function that lists the files modified by the commit.